dotcom-rendering/src/components/marketing/epics/ContributionsLiveblogEpic.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import { SecondaryCtaType } from '@guardian/support-dotcom-components';
+import type { Meta, StoryObj } from '@storybook/react';
+import lzstring from 'lz-string';
+import React from 'react';
+import { ContributionsLiveblogEpic } from './ContributionsLiveblogEpic';
+import { props } from './utils/storybook';
+
+const { variant, articleCounts, tracking } = props;
+
+type WithJsonProps<T> = T & { json?: string };
+type Props = WithJsonProps<
+	React.ComponentProps<typeof ContributionsLiveblogEpic>
+>;
+const meta: Meta<Props> = {
+	component: ContributionsLiveblogEpic,
+	title: 'Components/marketing/ContributionsLiveblogEpic',
+	args: {
+		variant,
+		articleCounts,
+		tracking,
+		countryCode: 'GB',
+		json: '',
+	},
+	render: ({ json, ...args }) => {
+		const jsonProps = json
+			? JSON.parse(lzstring.decompressFromEncodedURIComponent(json))
+			: {};
```

Comment:
We accept the props in the url to enable the RRCP to iframe the DCR storybook, for previewing - https://github.com/guardian/support-admin-console/pull/533